server/prometheus.go

```diff
@@ -0,0 +1,27 @@
+package server
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/prebid/prebid-server/config"
+	metricsconfig "github.com/prebid/prebid-server/pbsmetrics/config"
+	prometheusMetrics "github.com/prebid/prebid-server/pbsmetrics/prometheus"
+)
+
+func newPrometheusServer(cfg *config.Configuration, metrics *metricsconfig.DetailedMetricsEngine) *http.Server {
+	var proMetrics *prometheusMetrics.Metrics
+
+	proMetrics = metrics.PrometheusMetrics
+
+	if proMetrics == nil {
+		glog.Fatal("Prometheus metrics configured, but a Prometheus metrics engine was not found. Cannot set up a Prometheus listener.")
+	}
+	return &http.Server{
+		Addr:    cfg.Host + ":" + strconv.Itoa(cfg.Metrics.Prometheus.Port),
```

Comment:
This port needs to be exposed in the `Dockerfile`. Otherwise outside stuff (like Prometheus) won't be able to get at it.